packages/widgets/src/blocks/widget-box/edit.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	__experimentalUseInnerBlocksProps as useInnerBlocksProps,
+	useBlockProps,
+	InnerBlocks,
+	RichText,
+} from '@wordpress/block-editor';
+import { __ } from '@wordpress/i18n';
+import { createBlock } from '@wordpress/blocks';
+
+export default function Edit( { attributes, setAttributes, clientId } ) {
+	const innerBlocksProps = useInnerBlocksProps(
+		{
+			className: 'wp-widget-box__inner-blocks',
+		},
+		{
+			renderAppender: InnerBlocks.ButtonBlockAppender,
+		}
+	);
+
+	const blockProps = useBlockProps();
+
+	return (
+		<div { ...blockProps }>
+			<RichText
+				className="widget-title"
+				tagName="h2"
+				aria-label={ __( 'Widget title' ) }
+				placeholder={ __( 'Add a Widget title' ) }
+				value={ attributes.widgetTitle }
```

Comment:
Is it `widgetTitle` or is it `name`?

Reply:
I think `title` because that's the class used in the current widgets implementation. Also I think most legacy widgets use "title". I could be wrong. Why do you think `name` is more appropriate?

Reply:
It should be `name` because that's what is declared in the attributes in `block.json`. Currently, when adding a title, saving and reloading the widgets editor, the block becomes invalid because the title isn't being saved.

Reply:
Yeh but that attribute is entirely arbitary. I just carried it across due to copy/paste 😄 

I believe it should be `title` because that is the terminology used in legacy Widget's (e.g. https://developer.wordpress.org/reference/hooks/widget_title/).

Is there a reason other than what I accidentally named the attr in`block.json` why `name` would be preferable?

Reply:
Updated to use `title` as the attribute.